pkg/debugapi/postage.go

```diff
@@ -381,3 +379,57 @@ func (s *Service) postageTopUpHandler(w http.ResponseWriter, r *http.Request) {
 		BatchID: id,
 	})
 }
+
+func (s *Service) postageDiluteHandler(w http.ResponseWriter, r *http.Request) {
+	idStr := mux.Vars(r)["id"]
+	if idStr == "" || len(idStr) != 64 {
```

Comment:
I think the first check is redundant, if it's empty string it will fail the len check anyway

Reply:
This is how we do it in other API endpoints as well. The reasoning behind it, AFAIK:

Empty string check ` x == ""` is far more efficient than `len()` check. If in the `||` the first condition is true, the second one is not evaluated at all. This is why I would prefer having this check and evaluating empty strings slightly early.